Makefile

```diff
@@ -42,3 +42,14 @@ cross-images:
 		docker build --tag quickwit/cross:$$tag --file ./build/cross-images/$$tag.dockerfile ./build/cross-images; \
 		docker push quickwit/cross:$$tag; \
 	done
+
+# TODO: to be replaced by https://github.com/quickwit-inc/quickwit/issues/237
+TARGET ?= x86_64-unknown-linux-gnu
+.PHONY: build
+build:
+	@echo "Building binary for target=${TARGET}"
+	if [ "${TARGET}" = "x86_64-unknown-linux-musl" ]; then \
+        cross build --release --features release-feature-set --target ${TARGET}; \
```

Comment:
nit: can you fix the indentation?

Reply:
Just wondering what could be the origin of this, my local branch is fine and git says it's in sync with remote. 🤔

Reply:
Maybe it's the GitHub rendering. Don't worry too much about it.

Reply:
It was my vscode, not Gihub